cmd/application_remote.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var remoteName, remoteURL string
+
+var appRemoteCmd = &cobra.Command{
+	Use:     "remote",
+	Short:   "Add a remote to your current git repository",
+	Example: "civo app remote APP_NAME -r REMOTE_NAME -u REMOTE_URL",
```

Comment:
I think the URL should be autogenerated.

Reply:
Remote URL should be : git@git.civo.app:<region>/<accountID>/<app-name>

Reply:
How will we get the Account ID from the API key?